galaxy_ng/app/access_control/statements/roles.py

```diff
@@ -1,67 +1,112 @@
-VIEWSETS = {
-    "CollectionViewSet": {
-        "LOCKED_ROLES": {
-            "galaxy.collection_admin": [
-                "galaxy.change_namespace",
-                "galaxy.delete_namespace",
-                "galaxy.view_namespace",
-                "galaxy.upload_to_namespace",
-                "ansible.delete_collection",
-            ],
-        }
+LOCKED_ROLES = {
+    "galaxy.collection_admin": {
+        "permissions": [
+            "galaxy.add_namespace",
+            "galaxy.change_namespace",
+            "galaxy.delete_namespace",
+            "galaxy.upload_to_namespace",
+            "ansible.delete_collection",
+            "ansible.change_collectionremote",
+            "ansible.view_collectionremote",
```

Comment:
Today the `system:partner-engineering` group has these perms https://github.com/ansible/galaxy_ng/blob/b6eb764d54c6e7a57bb391d77aee4da06e339b5a/galaxy_ng/app/management/commands/maintain-pe-group.py#L27

The updated management command in this feature branch keeps those perms via these roles https://github.com/ansible/galaxy_ng/blob/6bead6e86732cdff7bd8329003bec4a3c6916707/galaxy_ng/app/management/commands/maintain-pe-group.py#L28-L33

This PR will give this group `ansible.change_collectionremote` and `ansible.view_collectionremote` which is probably ok since those viewsets would be unavailable.